.gitignore

```diff
@@ -52,3 +52,12 @@ vendor
 #################
 .env
 .vscode
+
+# PHPUnit #
+###########
+build/
+.phpunit.result.cache
+
+# Javascript/Node stuff #
+#########################
+node_modules/
```

Comment:
You know, this is an unrelated change... I am going to let it slide but I have to comment...